spec/system/school/courses/exports_spec.rb

```diff
@@ -170,10 +210,47 @@
     expect(page).to have_text("Included Inactive Students")
   end
 
+  scenario "school admin creates a student export with user standings" do
+    sign_in_user school_admin.user, referrer: exports_school_course_path(course)
+
+    find("h5", text: "Create New Export").click
+    find(".toggle-button__group .toggle-button__button", text: "Yes").click
+
+    click_button("Create Export")
+
+    expect(page).to have_text("Your export is being processed")
+
+    export = CourseExport.last
+    expect(export.include_user_standings).to eq(true)
+    expect(page).to have_text("Includes user standings")
+  end
+
+  scenario "school admin creates a student export without user standings" do
+    sign_in_user school_admin.user, referrer: exports_school_course_path(course)
+
+    find("h5", text: "Create New Export").click
+    find(".toggle-button__group .toggle-button__button", text: "No").click
+
+    click_button("Create Export")
+
+    expect(page).to have_text("Your export is being processed")
+    expect(CourseExport.last.include_user_standings).to eq(false)
+  end
```

Comment:
This can be a one-line addition to the first spec, which tests creation of an export without choosing any option.